File: Belah Ketupat (Fill)/Java/belah-ketupat-fill-i.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
	    
	    Scanner userInput = new Scanner(System.in);
    	
        System.out.print("Masukan Lebar Belah Ketupat : ");
        int x = userInput.nextInt();
        for(int i = 1; i <= x; i++){
            for(int j = x; j >= i; j--){
                System.out.print(" ");
            }
            
            for(int l = 2; l <= i; l++){
                System.out.print("*");
            }
            
            for(int k = 1; k <= i; k++){
                    System.out.print("*");
                }
            System.out.println("");
        }
        
        for(int i = 1; i <= (x - 1); i++){
            for(int j = x; j <= (i + x); j++){
                System.out.print(" ");
            }
            
            for(int l = 0; l <= (x - i - 1); l++){
                System.out.print("*");
            }
            
            for(int k = 0; k <= x-i-2; k++){
                System.out.print("*");
            }
            System.out.println("");
        }
	}
}

File: Belah Ketupat (Fill)/C++/belah-ketupat-fill-i.cpp
#include <iostream>
using namespace std;

int main(){
    int x;
    cout << "Masukan Lebar Belah Ketupat : ";
    cin >> x;
    for(int i = 1; i <= x; i++){
        for(int j = x; j >= i; j--){
            cout<<" ";
        }
        
        for(int l = 2; l <= i; l++){
            cout<<"*";
        }
        
        for(int k = 1; k <= i; k++){
                cout<<"*";
            }
        cout<<endl;
    }
    
    for(int i = 1; i <= (x - 1); i++){
        for(int j = x; j <= (i + x); j++){
            cout<<" ";
        }
        
        for(int l = 0; l <= (x - i - 1); l++){
            cout<<"*";
        }
        
        for(int k = 0; k <= x-i-2; k++){
                cout<<"*";
            }
        cout<<endl;
    }
    return 0;
}
